.github/workflows/docker-build.yml

```diff
@@ -61,6 +61,7 @@ jobs:
             type=ref,event=pr
             type=semver,pattern={{version}}
             type=semver,pattern={{major}}.{{minor}}
+            type=raw,value=latest,enable={{is_default_branch}}
```

Comment:
This adds the `latest` tag to images build from `main`. They are already tagged by the branch name, i.e. `main`. It may be sensible to keep the original behaviour, i.e. have the `latest` tag point to the latest versioned release.

Reply:
removed